### DISCUSSION TITLE
Logout from SOC no longer redirects to login page

### DISCUSSION THREAD

**Discussion by stijnpieters** [Security-Onion-Solutions/securityonion#2448] [Unsupported Versions]
Today I updated from 2.3.10 to 2.3.21. 

It seems in the new version logging out no longer redirects me to the login page of SOC. Refreshing after clicking logout does redirect. 

Maybe related to the new alerts interface way of refreshing the page content?

**Comment by dougburks** [comment]
I just tested this a few times on a few different installations and it appears to be working correctly for me. Have you tried purging browser cache, incognito mode, or a different browser altogether to see if the behavior is any different?
